src/test/java/baseClasses/FBPageTest.java

```diff
@@ -23,6 +26,15 @@ public void demoTest()
 		driver.close();
 	}
 	
+	

```

Comment:
Add this test case to the Test annotation and add to your testing.xml file as well.